spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature 'sign in' do
+  let(:user) do
+    Fabricate(:user, email: 'user@timeoverflow.org', password: 'papapa22')
+  end
+
+  context 'with a valid password' do
+    it 'signs the user in' do
+      expect(Capybara.current_session.driver.browser.manage.cookie_named('_timeoverflow_session')).to be_falsy
+
+      sign_in_with(user.email, 'papapa22')
+
+      expect(Capybara.current_session.driver.browser.manage.cookie_named('_timeoverflow_session')).to be_truthy
+    end
```

Comment:
I would also expect to navigate to a page which contains the nav menu, or the terms of service